hw/bsp/hifive1/riscv_openocd.cfg

```diff
@@ -0,0 +1,34 @@
+adapter_khz     10000
```

Comment:
The same comment regarding license blurb.

Thanks for doing this as ftdi openocd device. It's great to have this as an example in one of the BSPs.
Do you mind including the wiring info in text form either in this file, or README.md within this directory?